plugins/webhooks/index.js

```diff
@@ -232,6 +237,21 @@ async function createPREvents(options, request) {
     return Promise.all(events);
 }
 
+/**
+ * Resolve ChainPR flag
+ * @method resolveChainPR
+ * @param  {Boolean}  chainPR              Plugin Chain PR flag
+ * @param  {Pipeline} pipeline             Pipeline
+ * @param  {Object}   pipeline.annotations Pipeline-level annotations
+ * @return {Boolean}
+ */
+function resolveChainPR(chainPR, pipeline) {
+    const defaultChainPR = typeof chainPR === 'undefined' ? false : chainPR;
+    const annotChainPR = pipeline.annotations[ANNOT_CHAIN_PR];
+
+    return typeof annotChainPR === 'undefined' ? defaultChainPR : annotChainPR;
```

Comment:
[ask] Is there any case that value of chainPR becomes `undefined`?

Reply:
Yes, when there is no `screwdriver.cd/chainPR` annotation in a `screwdriver.yaml`.